tests/test_pytorch_lightning.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import annotations
+
+from collections.abc import Sequence
+
+import numpy as np
+import pytest
+
+import optuna
+from optuna._imports import try_import
+from optuna.testing.pruners import DeterministicPruner
+from optuna.testing.storages import StorageSupplier
+from optuna.trial import Trial
+from optuna.trial import TrialState
+
+from optuna_integration import PyTorchLightningPruningCallback
```

Comment:
This line causes the following error.
```sh
    from optuna_integration import PyTorchLightningPruningCallback
E   ImportError: cannot import name 'PyTorchLightningPruningCallback' from 'optuna_integration'
```

To avoid the error, please use `from optuna_integration.pytorch_lightning import PyTorchLightningPruningCallback` instead.

FYI: another possible approach is introducing an elaborated `__init__.py` like [this](https://github.com/optuna/optuna/blob/master/optuna/integration/__init__.py).